Sources/Purchasing/Placements.swift

```diff
@@ -0,0 +1,29 @@
+//
+//  Copyright RevenueCat Inc. All Rights Reserved.
+//
+//  Licensed under the MIT License (the "License");
+//  you may not use this file except in compliance with the License.
+//  You may obtain a copy of the License at
+//
+//      https://opensource.org/licenses/MIT
+//
+//  Placements.swift
+//
+//  Created by Guido Torres on 15/02/2024.
+
+import Foundation
+
+internal final class Placements: NSObject {
```

Comment:
I'm wondering if this should be inside `Offering` as well, so it's clearer it's refering to that. Right now this is a bit too generic IMO